src/modules/mim/firewall.json

```diff
@@ -36,102 +36,259 @@
           "schema": "string"
         },
         {
-          "name": "desiredFirewallRule",
+          "name": "configuredState",
           "type": "mimObject",
-          "desired": true,
+          "desired": false,
+          "schema": {
+            "type": "enum",
+            "valueSchema": "string",
+            "enumValues": [
+              {
+                "name": "unknown",
+                "enumValue": "unknown"
+              },
+              {
+                "name": "success",
```

Comment:
>success [](http://example.com/codeflow?start=25&length=7)

This is a state, so "succeeded" and "failed". 
Or: change "configuredState" to "configurationStatus" and keep these